README.md

```diff
@@ -1,2 +1,46 @@
 # getting-started-.net-maui-popup
 How to get started with .net maui popup
+
+The [.NET MAUI Popup](https://www.syncfusion.com/maui-controls/maui-popup?utm_source=github&utm_medium=listing&utm_campaign=maui-popup-github-samples) control displays an alert message with customized buttons or a desired view inside a pop-up. Users can completely customize the pop-up’s appearance and functionality, including its header, body, and footer.
```

Comment:
Removed this line
![image](https://github.com/SyncfusionExamples/getting-started-.net-maui-popup/assets/95272441/e38dfcc7-6f46-48f5-9192-d2ca4f962534).

Use the below instaed..
A quick-start project that helps you integrate the Popup control with the .NET MAUI application by adding the  Syncfusion MAUI ListView package from NuGet. This project also explains on how to customize the pop-up’s appearance and functionality, including its header, body, and footer.

Reply:
Done